holidays/locale/en_US/LC_MESSAGES/TW.po

```diff
@@ -0,0 +1,58 @@
+# Taiwan holidays en_US localization.
+# Authors: PPsyrius <ppsyrius@ppsyrius.dev>, (c) 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Python Holidays 0.40\n"
+"POT-Creation-Date: 2023-11-24 16:16+0700\n"
+"PO-Revision-Date: 2023-12-02 11:31+0700\n"
+"Last-Translator: PPsyrius <ppsyrius@ppsyrius.dev>\n"
+"Language-Team: Python Holidays Translation Team\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Generated-By: Lingua 4.15.0\n"
+"X-Generator: Poedit 3.4.1\n"
+
+#. %s (Observed).
+#, c-format
+msgid "%s (慶祝)"
+msgstr "%s (Observed)"
+
+#. Founding Day of the Republic of China.
+msgid "中華民國開國紀念日"
+msgstr "Founding Day of the Republic of China"
+
+#. Chinese New Year's Eve.
+msgid "農曆除夕"
+msgstr "Chinese New Year's Eve"
+
+#. Chinese New Year (Spring Festival).
+msgid "春節"
+msgstr "Chinese New Year (Spring Festival)"
```

Comment:
Maybe just "Chinese New Year" (as on DGPA site)?

Reply:
Yeah, that might be for the better if we go all the way with standardization - fixed now.